Xbim.WindowsUI.sln

```diff
@@ -29,29 +30,48 @@ EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|x86 = Debug|x86
 		Release|Any CPU = Release|Any CPU
+		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{89F98E53-DE4E-4427-8244-D5934AD08849}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{89F98E53-DE4E-4427-8244-D5934AD08849}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{89F98E53-DE4E-4427-8244-D5934AD08849}.Debug|x86.ActiveCfg = Debug|x86
```

Comment:
x86 allows building/testing on Arm64 (until we get a GE & Xbim.Geometry.Occt with Arm64 support)